js/dropdowns.js

```diff
@@ -106,14 +113,69 @@ Dropdown.prototype.handleCheck = function(e) {
   }
 };
 
+Dropdown.prototype.handleDropdownItemClick = function(e) {
+  var $button = $(e.target);
+  var $input = $('#' + $button.data('label'));
```

Comment:
Let's scope this to `this.$selected`. So: `this.$selected.find('#' + $button.data('label'));`